libs/brain/select/src/lib/brn-select-trigger.directive.ts

```diff
@@ -43,11 +55,20 @@ export class BrnSelectTriggerDirective<T> implements AfterViewInit, OnDestroy {
 	});
 
 	private _resizeObserver?: ResizeObserver;
+	private statusChangedSubscription?: Subscription;
 
 	constructor() {
 		this._select.trigger.set(this);
 	}
 
+	ngOnInit() {
+		if (this._ngControl) {
+			this.statusChangedSubscription = this._ngControl.statusChanges?.subscribe(() => {
+				this._changeDetector.markForCheck();
+			});
+		}
```

Comment:
This allows to apply the correct class based on the ngControl valid or invalid state.
@goetzrobin @ashley-hunter I'm not sure if this is the best approach in this case to reevaluate `_ngControl?.invalid` and `_ngControl?.valid`.

Reply:
This is exactly what I was going to suggest 😊 if it works I have no issues with it.

Reply:
Tests are green again locally, let's see how happy CI will be with the select test 🤣